_scripts/build.bash

```diff
@@ -29,5 +29,4 @@ bash _scripts/runPreprocessor.bash execute --debug=all --norun=$norun $skipcache
 # Main site
 cd hugo
 $time npm ci
-$time npm run icons
-$time hugo --cleanDestinationDir $@
+$time hugo --cleanDestinationDir --minify $@
```

Comment:
I'd suggest setting a variable for this above (like `time`, empty by default), such that minification only happens as part of the Netlify run. Then use that variable as the argument.